core/src/main/java/org/apache/iceberg/MetricsConfig.java

```diff
@@ -92,7 +92,7 @@ public static MetricsConfig forPositionDelete(Table table) {
     ImmutableMap.Builder<String, MetricsMode> columnModes = ImmutableMap.builder();
 
     columnModes.put(MetadataColumns.DELETE_FILE_PATH.name(), MetricsModes.Full.get());
-    columnModes.put(MetadataColumns.DELETE_FILE_POS.name(), MetricsModes.Full.get());
+    columnModes.put(MetadataColumns.DELETE_FILE_POS.name(), MetricsModes.Counts.get());
```

Comment:
I am debating here. These boundaries were never used anywhere, I see no value in these during planning. I also doubt there would be any value in readers given how we apply deletes. Can be convinced otherwise.